README.md

```diff
@@ -20,25 +20,43 @@ The following commands are supported for both Maven and Gradle Quarkus projects:
   * `Quarkus: Add extensions to current project`: Add Quarkus extensions to currently opened Quarkus project
   * `Quarkus: Debug current Quarkus project`: Launches the Maven `quarkus:dev` plugin or the Gradle `quarkusDev` command and automatically attaches a debugger  
 
-## Quarkus `application.properties` Features
-  * Completion support for Quarkus properties
-  * Hover support for Quarkus properties
-  * Validation support for Quarkus properties 
-  * Support for Quarkus profiles
-  * Outline support (flat or tree view)
+## Quarkus / MicroProfile `properties` Features
 
-## Quarkus code snippets
-This extension provides several code snippets, available when editing Java files:
+In `application.properties` and `microprofile-config.properties` files, you will benefit with:
 
-  * **qrc** - Create a new Quarkus resource class
-  * **qrm** - Create a new Quarkus resource method
-  * **qtrc** - Create a new Quarkus test resource class
-  * **qntrc** - Create a new Quarkus native test resource class
+  * Completion support for Quarkus / MicroProfile properties
+  * Hover support for Quarkus / MicroProfile properties
+  * Validation and Quickfixes support for Quarkus / MicroProfile properties 
+  * Support for Quarkus profiles
+  * Outline support (flat or tree view)
+  * Code snippets:
+    * **qds** - Configure a Quarkus datasource
+    * **qj** - Configure a Jaeger tracer
+  
+## Quarkus `yaml` Features
 
-When editing `application.properties` files, you have access to:
+In `application.yaml`, `application.yml` files, you will benefit with:
```

Comment:
In `application.yaml` and `application.yml` files

Reply:
fixed